### PR TITLE
[CI ONLY] Automatically nightly triggered master build statistics

### PR DESCRIPTION
**Pull request by chibenwa**
Launches 5 master builds per night.

In the morning, we need to inspect the results, and consolidate weekly statistics about master builds.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Apache James Project
 ====================
 
+Simple change
```
Comment (chibenwa) [inline README.adoc]: Comment

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (openpaas-james) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

Comment (chibenwa) [reply README.adoc]: Automatic test this please

**Comment by chibenwa** [comment]
Abandon this

**Pull request closed without merging**